### ISSUE TITLE
filter-idempotent

### ISSUE THREAD

**Issue by mechvel** [agda/agda-stdlib#940]
Probably this is needed in standard library:
``` 
filter-idempotent :  ∀ (p? : Decidable P) →  filter p? ∘ filter p? ≗ filter p? 
``` 
- ?

**Comment by gallais** [comment]
Could we avoid opening a new issue for every single lemma related to filter?
They can very well be accumulated in a single issue as a TODO list with checkboxes
(`* [ ] If you write this, the description will be preceded by a checkbox`).

**Comment by MatthewDaggitt** [comment]
Closed in favour of #933
